static/app/components/onboarding/footerWithViewSampleErrorButton.tsx

```diff
@@ -0,0 +1,397 @@
+}: {
+  clientState: ReturnType<typeof usePersistedOnboardingState>[0];
+  nextLocation: Location;
+  router: RouteComponentProps<{}, {}>['router'];
+  setClientState: ReturnType<typeof usePersistedOnboardingState>[1];
+}) {
+  const mod = await import('sentry/components/onboarding/changeRouteModal');
+
+  const {ChangeRouteModal} = mod;
+
+  openModal(deps => (
+    <ChangeRouteModal
+      {...deps}
+      router={router}
+      nextLocation={nextLocation}
+      clientState={clientState}
+      setClientState={setClientState}
+    />
+  ));
+}
+
+export function FooterWithViewSampleErrorButton({
+  projectSlug,
+  projectId,
+  router,
+  newOrg,
+}: Props) {
+  const organization = useOrganization();
+  const preferences = useLegacyStore(PreferencesStore);
+  const [firstError, setFirstError] = useState<string | null>(null);
+  const [firstIssue, setFirstIssue] = useState<Group | undefined>(undefined);
+  const [clientState, setClientState] = usePersistedOnboardingState();
+  const {projects} = useProjects();
+  const onboardingContext = useContext(OnboardingContext);
+  const projectData = projectId ? onboardingContext.data[projectId] : undefined;
+  const selectedProject = projects.find(project => project.slug === projectSlug);
+
+  useQuery<Project>([`/projects/${organization.slug}/${projectSlug}/`], {
+    staleTime: 0,
+    refetchInterval: DEFAULT_POLL_INTERVAL,
+    enabled:
+      !!projectSlug && !firstError && projectData?.status === OnboardingStatus.WAITING, // Fetch only if the project is available and we have not yet received an error,
+    onSuccess: data => {
+      setFirstError(data.firstEvent);
+    },
+  });
+
+  // Locate the projects first issue group. The project.firstEvent field will
+  // *not* include sample events, while just looking at the issues list will.
+  // We will wait until the project.firstEvent is set and then locate the
+  // event given that event datetime
+  useQuery<Group[]>([`/projects/${organization.slug}/${projectSlug}/issues/`], {
+    staleTime: 0,
+    enabled:
+      !!firstError && !firstIssue && projectData?.status === OnboardingStatus.PROCESSING, // Only fetch if an error event is received and we have not yet located the first issue,
+    onSuccess: data => {
+      setFirstIssue(data.find((issue: Group) => issue.firstSeen === firstError));
+    },
+  });
+
+  useEffect(() => {
+    if (!projectId || !!projectData) {
+      return;
+    }
+
+    onboardingContext.setProjectData({
+      projectId,
+      projectSlug,
+      status: OnboardingStatus.WAITING,
+    });
+  }, [projectData, onboardingContext, projectSlug, projectId]);
+
+  useEffect(() => {
+    if (!projectId) {
+      return;
+    }
+
+    if (!firstError) {
+      return;
+    }
+
+    if (projectData?.status !== OnboardingStatus.WAITING) {
+      return;
+    }
+
+    trackAdvancedAnalyticsEvent('onboarding.first_error_received', {
+      organization,
+      new_organization: !!newOrg,
+      project_slug: projectSlug,
```

Comment:
@priscilawebdev we should track the `project_id` instead because we track that in other places as well. Would also be nice to track the platform.

Reply:
I realize we already track `peroject_slug` but that was definitely a mistake, we should switch to `project_id` for all of the analytics

Reply:
ok will add that now...thanks for the feedback 🙏

Reply:
https://github.com/getsentry/sentry/pull/45922 😉